### PR TITLE
[WIP] split reader and library in renderer folder

### PR DESCRIPTION
**Pull request by panaC**

### CODE REVIEW

File: src/renderer/library/components/dialog/publicationInfos/TagManager.tsx [hunk 1]
```diff
@@ -235,15 +174,16 @@ const mapStateToProps = (state: RootState) => ({
 });
 
 const mapDispatchToProps = (dispatch: TDispatch, _props: IBaseProps) => ({
-    setTags: (pubId: string, publication: TPublication, tagsName: string[], tagsOpds: string[] | IOpdsTagView[]) => {
+    setTags: (pubId: string, publication: TPublication, tagsName: string[]) => {
         apiDispatch(dispatch)()("publication/updateTags")(pubId, tagsName);
         dispatch(
             dialogActions.updateRequest.build<DialogTypeName.PublicationInfoLib>(
                 {
+                    // @ts-ignore
```
Comment (panaC) [inline src/renderer/library/components/dialog/publicationInfos/TagManager.tsx]: FIXME

File: src/renderer/common/components/dialog/publicationInfos/publicationInfoContent.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import { FormatPublicationLanguage } from "./formatPublicationLanguage";
+import { FormatPublisherDate } from "./formatPublisherDate";
+import LcpInfo from "./LcpInfo";
+import PublicationInfoDescription from "./PublicationInfoDescription";
+
+export interface IProps {
+    publication: TPublication;
+    toggleCoverZoomCb: (coverZoom: boolean) => void;
+    ControlComponent?: React.ComponentType;
+    TagManagerComponent: any;
```
Comment (panaC) [inline src/renderer/common/components/dialog/publicationInfos/publicationInfoContent.tsx]: FIXME

File: src/renderer/library/components/dialog/publicationInfos/TagManager.tsx (hunk 1 above)
Comment (panaC) [reply src/renderer/library/components/dialog/publicationInfos/TagManager.tsx]: fixed

File: src/renderer/common/components/dialog/publicationInfos/publicationInfoContent.tsx (hunk 2 above)
Comment (panaC) [reply src/renderer/common/components/dialog/publicationInfos/publicationInfoContent.tsx]: fixed

File: webpack.config.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ const util = require('util');
 
 const mainConfig = require("./webpack.config.main");
 const rendererConfig = require("./webpack.config.renderer");
```
Comment (danielweck) [inline webpack.config.js]: Replace: `"./webpack.config.renderer"` with `"./webpack.config.renderer"-library`

Comment (panaC) [reply webpack.config.js]: fixed

**Pull request merged**